README.md

```diff
@@ -39,6 +39,30 @@ The following is an abbreviated guide. Check out [The Marker Book] for detailed
 
 ### Installation
 
+#### Download pre-compiled binaries (recommended)
```

Comment:
This should also be duplicated to `cargo-marker`'s readme.

---

Once the next release is reached and I have a bit more time, I might ask CatGBT[^1] for a script that ensures that two regions in different files remain in sync 

[^1]: Okay I think that is my favorite misspelling of the month, and I'll stick with it ^^

Reply:
Hm, maybe `cargo-marker`'s readme could be a symlink to the root README? I think people actually don't read the docs on crates.io, they just go straight to the repo's docs (at least I do).

Reply:
That might be a good option, why are close enough. The main README contains a bit more content, but it wouldn't be bad to have that in `cargo-marker`'s readme as well :)

Do you want to add the link in this PR?

Reply:
I see that the `README.md` for `cargo-marker` details a bit more about what `cargo-marker` is, and I wouldn't like to lose all that information overwriting this file with a symlink to the root README. So I re-considered this and added a reference to the main repo readme file for details on the installation.

Did you mean adding a URL link as well, right?

Reply:
I meant a file link, I didn't remember that the README had more details. In that case, a URL link should be sufficient. Now that we have precompiled binaries, it's likely that users will take that route and not install Marker from crates.io anyways :)